ui/core/src/config.testnet.json

```diff
@@ -1,15 +1,15 @@
 {
   "sifAddrPrefix": "sif",
-  "sifApiUrl": "https://api-testnet-042.sifchain.finance",
-  "sifWsUrl": "wss://rpc-testnet-042.sifchain.finance/websocket",
-  "sifRpcUrl": "https://rpc-testnet-042.sifchain.finance",
-  "sifChainId": "sifchain-testnet-1",
+  "sifApiUrl": "https://api-testnet-042-ibc.sifchain.finance",
+  "sifWsUrl": "wss://rpc-testnet-042-ibc.finance/websocket",
+  "sifRpcUrl": "https://rpc-testnet-042-ibc.sifchain.finance",
+  "sifChainId": "sifchain-testnet-042-ibc",
   "web3Provider": "metamask",
   "nativeAsset": "rowan",
   "cryptoeconomicsUrl": "https://api-cryptoeconomics-devnet.sifchain.finance/api",
   "bridgebankContractAddress": "0x6CfD69783E3fFb44CBaaFF7F509a4fcF0d8e2835",
   "keplrChainConfig": {
-    "chainName": "SifchainTestnet-042",
+    "chainName": "SifchainTestnet-IBC",
```

Comment:
Is that correct chain name?

Reply:
Yea, `chainName` in this context is arbitrary display name. `sifChainId` above is what's used